<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
}

QFrame
{
background: white;
border-radius: 1px;
}

#frame_2
{
}

QLineEdit
{
border:none ;
color: black;
border-bottom: 1px solid orange;

background: transparent;
}

QLabel
{
font-size: 13px;
color:  black;
}

#label_3
{
font-size: 20px;
color: #256aff;
}

QPushButton
{
border-radius: 1px;
font-size: 16px;
font-color: white;
}


QTabBar::tab 
{
	background-color: white;
	border-top-left-radius: 0px;
	border-top-right-radius: 0px;

	min-width: 8ex;
	padding: 5px;
}

QTabBar::tab:selected
 {
	color: white;
	background-color: black;
}

QTabBar::tab:!selected
 {
	margin-top: 0px;
	background: #c6c6c6;
	color: black
}

QTabWidget::tab-bar 
{
    alignment: center;
}
#label_7
{
font-size: 20px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>430</width>
      <height>430</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>321</width>
       <height>331</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Register</string>
      </attribute>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>321</width>
         <height>301</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>120</y>
          <width>221</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>70</y>
          <width>221</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>211</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Enter username:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>100</y>
          <width>211</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enter password:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>212</y>
          <width>261</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:  orange;
color: white;</string>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent;
color: black;
</string>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>168</y>
          <width>221</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>148</y>
          <width>211</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Repeat password:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>269</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Login</string>
      </attribute>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>321</width>
         <height>301</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>44</x>
          <y>140</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>43</x>
          <y>80</y>
          <width>221</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_username">
        <property name="geometry">
         <rect>
          <x>43</x>
          <y>60</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_pass">
        <property name="geometry">
         <rect>
          <x>45</x>
          <y>120</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_login">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>193</y>
          <width>261</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:  orange;
color: white;</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent;
color: black;</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_result">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>250</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: orange;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
